webapi/Extensions/ISemanticMemoryClientExtensions.cs

```diff
@@ -54,7 +54,10 @@ public static void AddSemanticMemoryServices(this WebApplicationBuilder appBuild
             }
         }
 
-        IKernelMemory memory = memoryBuilder.FromAppSettings().Build();
+        IKernelMemory memory = memoryBuilder.FromConfiguration(
```

Comment:
We have to provide our own config instead of letting the kernel memory builder decide what to read.